Source/WebKit/UIProcess/ProcessThrottler.h

```diff
@@ -143,6 +143,7 @@ class ProcessThrottler : public CanMakeWeakPtr<ProcessThrottler> {
     void setAllowsActivities(bool);
     void setShouldTakeSuspendedAssertion(bool);
     void delaySuspension();
+    bool isSuspended() const { return !m_assertion; }
```

Comment:
Patch looks good but can you please confirm that `isSuspended()` will not return true unconditionally for non-Cocoa ports? We wouldn't want to disable responsiveness checks on GTK port for example.